testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/modcluster/StartWorkersInSuspendedModeTestCase.java

```diff
@@ -0,0 +1,133 @@
+        return pojoWar;
+    }
+
+    public StartWorkersInSuspendedModeTestCase() {
+        super(new String[] { NODE_1, LOAD_BALANCER_1 }, new String[]{DEPLOYMENT_1});
+    }
+
+    @Test
+    public void testWorkerNodeIsRegisteredStopped() throws Exception {
+        ServerReload.executeReloadAndWaitForCompletion(TestSuiteEnvironment.getModelControllerClient(), ServerReload.TIMEOUT,
+                                                       false, true,
+                                                       null,
+                                                       -1, null);
+
+        assertWorkerNodeContextIsStopped();
+    }
+
+    private void assertWorkerNodeContextIsStopped() throws Exception {
+
+        ModelNode op = createOpNode("subsystem=undertow/configuration=filter/mod-cluster=load-balancer/balancer=mycluster/node=" + NODE_1,
+                                    READ_RESOURCE_OPERATION);
+        op.get(ADDRESS).add("context", "/" + MODULE_NAME);
+        op.get(RECURSIVE).set(true);
+        op.get(INCLUDE_RUNTIME).set(true);
+
+        final ModelControllerClient client = TestSuiteEnvironment.getModelControllerClient(
+           null,
+           TestSuiteEnvironment.getServerAddress(),
+           TestSuiteEnvironment.getServerPort() + LB_OFFSET);
+
+        // might need to wait for mod_cluster nodes to be registered
+        long start = System.currentTimeMillis();
+        ModelNode modelNode = null;
+        while (System.currentTimeMillis() - start < STATUS_REFRESH_TIMEOUT) {
+            modelNode = client.execute(op);
+            if (modelNode.has(RESULT) && modelNode.get(RESULT).has(STATUS)) {
+                break;
+            }
+            Thread.sleep(100);
+        }
+
+        Assert.assertEquals(SUCCESS, modelNode.get(OUTCOME).asString());
+        Assert.assertEquals("stopped", modelNode.get(RESULT).get(STATUS).asString());
+    }
+
+    static class ServerSetupTask extends CLIServerSetupTask {
+        public ServerSetupTask() {
+
+            /*
+            Set up balancer using deprecated proxy-list option in modcluster subsystem
+             */
+            this.builder
+               .node(NODE_1)
+               .setup("/subsystem=modcluster/mod-cluster-config=configuration:write-attribute(name=advertise,value=false)")
+               .setup("/socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=proxy1:add(host=localhost,port=8590)")
+               .setup("/subsystem=modcluster/mod-cluster-config=configuration:list-add(name=proxies,value=proxy1)")
+               .teardown("/subsystem=modcluster/mod-cluster-config=configuration:list-remove(name=proxies,value=proxy1)")
+               .teardown("/socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=proxy1:remove()")
+               .teardown("/subsystem=modcluster/mod-cluster-config=configuration:write-attribute(name=advertise,value=true)")
```

Comment:
The `/subsystem=modcluster/mod-cluster-config=configuration` is legacy resource – which still works due to  compatibility – but new tests should use current resources (i.e. `/subsystem=modcluster/proxy=default`).